src/changelogs/fetch_changelogs.py

```diff
@@ -152,6 +233,21 @@ def _process_local_changelogs(self, changelog_path):
             try:
                 with open(changelog_path + f'/{version}.txt', 'r', encoding='utf8') as f:
                     changelogs = f.read()
-                    self.changelog_lines[version] = changelogs
+                    self.changelogs[version] = changelogs
             except Exception:
                 print(f'Issue with {file}, skipping')
+
+
+def format_date(date):
+    """
+    Reformat mm/dd/yyyy or mm-dd-yyyy to yyyy_mm_dd
+    """
+    if '/' in date:
+        date = date.split('/')
+    elif '-' in date:
+        date = date.split('-')
+    else:
+        raise ValueError(f'Invalid date format {date}')
+
+    date = f'{date[2]}_{date[0]}_{date[1]}'
+    return date
```

Comment:
One of the dates is `2024_11_1`. Is that a fault of the raw file?

Reply:
This is the decompiled line: ```
"Citadel_PatchNotes_HeroLabs_hero_slork_3": "<b>11/1/2024</b>\t\t\t<li>Health per boon reduced from 41 to 36\t\t\t<li>Shotgun damage per pellet reduced from 4.5 to 4\t\t\t<li>Shotgun falloff start reduced from 18m to 16m\t\t\t<li>Scalding Spray weapon damage gain per sec reduced from 8% to 7%\t\t\t<li>Fathom's Bloodletting updated to have the same values as the latest version of Shiv's Bloodletting</li>",
    ```
    So I think it's correct. Recall its yyyy_mm_dd

Reply:
Sure its valve's fault. Can we have it read `2024_11_01`.

Reply:
Ah I couldn't tell that's what you meant originally. Pushed an addition to `format_date`.